README.md

```diff
@@ -10,8 +10,8 @@ The entry point of the framework is the `Service`. The `Service` uses `Component
 
 - service, which orchestrates everything
 - components and processors, which provide a abstraction of adding processing functionality to the service
-  - asynchronous message processing (RabbitMQ, Kafka)
-  - synchronous processing (HTTP)
+  - asynchronous message processing (RabbitMQ, Kafka, AWS SQS)
+  - synchronous processing (HTTP, AWS SNS)
```

Comment:
This refers to processing requests and Patron does not process SNS messages only publishes them with the new client.